.github/workflows/main.yml

```
@@ -933,7 +933,8 @@ jobs:
         run: |
           docker run --rm -i -v $PWD:/root/indy-sdk --name libindy $DOCKER_IMG_NAME_PYTHON \
                               bash -c '(cd $HOME/indy-sdk/wrappers/python && \
-                              PACKAGE_VERSION=${{ env.PUBLISH_VERSION }} python3.6 setup.py sdist && \
+                              echo "${{ env.PUBLISH_VERSION }}">> version.txt &&\
```

Overwrite rather than append.
```suggestion
                              echo "${{ env.PUBLISH_VERSION }}" > version.txt &&\
```